src/Pelagos/Bundle/AppBundle/Security/LoginFormAuthenticator.php

```diff
@@ -161,6 +173,14 @@ public function checkCredentials($credentials, UserInterface $user)
             throw new AuthenticationException('Too many login attempts');
         }
 
+        // Check for expired password.
+        $now = new \DateTime('now');
+        $expiration = $user->getPasswordEntity()->getModificationTimeStamp()->add(new \DateInterval($this->maxPwAge));
```

Comment:
What if maxPwAge is blank/null?

Reply:
Oh yeah, I want that to be the case for never expiring, blank or zero. Good catch.